PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidMobile.java

```diff
@@ -22,9 +22,9 @@
 
 public class PrebidMobile {
 
-    public static final int TIMEOUT_MILLIS = 2_000;
+    static final int TIMEOUT_MILLIS = 2_000;
```

Comment:
In this case we can use `private static final`

Reply:
I agree, I have just made the change